hivemind/optim/grad_scaler.py

```diff
@@ -1,12 +1,14 @@
 import contextlib
```

Comment:
- the changes in this file are necessary to support delay_optimizer_step
- checked this to work with examples/albert (CollaborativeOptimizer)
- checked this to work with hivemind.Optimizer for mingpt2.py